### PR TITLE
dlr

### PR DESCRIPTION
**Pull request by jonthegeek**
### Common Mistakes to Catch Fast:
- [x] Did you increment the version number?
- [x] Did you re-build and document the package at the very end to make sure all changes were picked up?
- [x] Does `goodpractice::gp()` come up clean?

### Tests:
- [x] Do you have tests for all of the corner cases you can think of?

### Documentation:
- [x] Are all functions documented, including arguments and return values?
- [x] Is there at least one example in the documentation of each exported function?
- [x] Did you update the README (or consciously decide not to do so)?

### Code Quality:
- [x] Are all parameters validated?
- [x] Is the parameter that's most likely to be piped into this function the first parameter?
- [x] Are all internal functions prepended with '.'?
- [x] Did you deal with all hacky things that you meant to address before PRing?

### Housekeeping:
- [x] Did you review the author checklist (especially things in bold)?

### Documentation:
- [x] Can you understand what this feature is supposed to do from its help?
- [x] Does the example seem like a typical use case?

### Code Quality:
- [x] Can you easily determine what each section of code does?

### Tests:
- [x] Can you build the package with your personal Rstudio setup?
- [x] Do the tests pass for you?
- [x] Can you think of any missing tests?

### CODE REVIEW

**Review by jonathanbratt — CHANGES_REQUESTED** [review]
Minor typo, and a question regarding `if` condition.

File: R/env.R [hunk 1]
```diff
@@ -86,42 +86,57 @@ assign(
...
+  # The populate step only runs once per session, loading the data
   # from disk.
-  .populate_env_lookup(cache_dir)
+  .populate_env_lookup()
 
   this_row <- .wikimorphemes_env$lookup[
     .wikimorphemes_env$lookup$word == word,
   ]
-  if (nrow(this_row)) {
+  if (!is.null(this_row) && nrow(this_row)) {
```
[the remark this answers is not in the document]
Comment (jonthegeek) [reply R/env.R:97]: See discussion on Slack. `&&` to the rescue!

**Review by jonathanbratt — APPROVED** [review]
looks good!
